examples/md-flexible/src/Simulation.cpp

```diff
@@ -372,6 +465,106 @@ void Simulation::calculateGlobalForces(const std::array<double, 3> &globalForce)
+  long vtk = accumulateTime(_timers.vtk.getTotalTime());
+  long initialization = accumulateTime(_timers.initialization.getTotalTime());
+  long total = accumulateTime(_timers.total.getTotalTime());
+  long thermostat = accumulateTime(_timers.thermostat.getTotalTime());
+  long haloParticleExchange = accumulateTime(_timers.haloParticleExchange.getTotalTime());
+  long migratingParticleExchange = accumulateTime(_timers.migratingParticleExchange.getTotalTime());
+
+  if (_domainDecomposition.getDomainIndex() == 0) {
+    auto maximumNumberOfDigits = std::to_string(total).length();
+    std::cout << "Measurements:" << std::endl;
+    std::cout << timerToString("Total                        ", total, maximumNumberOfDigits, total);
+    std::cout << timerToString("  Initialization             ", initialization, maximumNumberOfDigits, simulate);
+    std::cout << timerToString("  Simulate                   ", simulate, maximumNumberOfDigits, total);
+    std::cout << timerToString("  PositionUpdate             ", positionUpdate, maximumNumberOfDigits, simulate);
+    std::cout << timerToString("  Boundaries:                ", haloParticleExchange + migratingParticleExchange,
+                               maximumNumberOfDigits, simulate);
+    std::cout << timerToString("    HaloParticleExchange     ", haloParticleExchange, maximumNumberOfDigits, simulate);
+    std::cout << timerToString("    MigratingParticleExchange", migratingParticleExchange, maximumNumberOfDigits,
+                               simulate);
+    std::cout << timerToString("  ForceUpdateTotal           ", forceUpdateTotal, maximumNumberOfDigits, simulate);
+    std::cout << timerToString("    ForceUpdatePairwise      ", forceUpdatePairwise, maximumNumberOfDigits,
+                               forceUpdateTotal);
+    std::cout << timerToString("    ForceUdpateGlobalForces  ", forceUpdateGlobalForces, maximumNumberOfDigits,
+                               forceUpdateTotal);
+    std::cout << timerToString("    ForceUpdateTuning        ", forceUpdateTuning, maximumNumberOfDigits,
+                               forceUpdateTotal);
+    std::cout << timerToString("    ForceUpdateNonTuninng    ", forceUpdateNonTuning, maximumNumberOfDigits,
+                               forceUpdateTotal);
+    std::cout << timerToString("  VelocityUpdate             ", velocityUpdate, maximumNumberOfDigits, simulate);
+    std::cout << timerToString("  Thermostat                 ", thermostat, maximumNumberOfDigits, simulate);
+    std::cout << timerToString("  Vtk                        ", vtk, maximumNumberOfDigits, simulate);
+    std::cout << timerToString("One iteration                ", total / _iteration, maximumNumberOfDigits, simulate);
+
+    const long wallClockTime = _timers.total.getTotalTime();
+    std::cout << timerToString("Total wall-clock time        ", wallClockTime, std::to_string(wallClockTime).length(),
+                               wallClockTime);
```

Comment:
```suggestion
    const long wallClockTime = _timers.total.getTotalTime();
    std::cout << timerToString("Total wall-clock time        ", wallClockTime, std::to_string(wallClockTime).length(),
                               total);
```
Then we directly get an idea about the efficiency